lang/hr/index.md

```diff
@@ -0,0 +1,262 @@
+izdavanja nove verzije svakog ovisnog paketa). Ako su pak ovisnosti specificirane
+previše slobodno, sigurno ćete osjetiti posljedice verzijskog promiskuiteta
+(pretpostavke kompatibilnosti s više budućih verzija nego što je razumno).
+Pakao ovisnosti je slučaj gdje zaključavanje verzije i/ili verzijski promiskuitet
+sprječavaju lagano i sigurno napredovanje projekta.
+
+Kao rješenje tog problema predlažem jednostavan set pravila i zahtjeva
+koji će nalagati kako se dodjeljuju i povećavaju oznake verzija.
+Ova se pravila temelje, no nisu nužno i ograničena, na već postojećoj
+i raširenoj uobičajenoj praksi u projektima otvorenog i zatvorenog koda.
+Kako bi ovaj sustav funkcionirao, potrebno je prvo objaviti javni API. Možemo
+to primijeniti u dokumentaciji ili u samom kodu. U svakom slučaju, važno je da
+API bude jasan i precizan. Jednom kad identificiramo javni API, komuniciramo
+promjene u njemu kroz specificirana povećanja u oznaci/broju verzije. Uzmimo
+format verzije X.Y.Z (MANJA.VEĆA.ZAKRPA). Ispravci bugova koji ne utječu na API
+povećavaju oznaku zakrpe, unatrag-kompatibilni dodaci/promjene u APIju
+povećavaju manju verziju, a unatrag inkompatibilne promjene u APIju povećavaju
+veću verziju.
+
+Ovaj sam sustav nazvao "Semantičko verzioniranje". Unutar ove sheme, oznake
+verzije i način na koji se mijenjaju prenose određene informacije o kodu koji
+se nalazi ispod i što se mijenjalo od jedne do druge verzije.
+
+
+Specifikacija semantičkog verzioniranja (SemVer)
+------------------------------------------
+
+Ključne riječi "MORA", "NE SMIJE", "POTREBNO", "BITI ĆE", "NEĆE BITI", "TREBALO BI",
+"NE BI TREBALO", "PREPORUČENO", "MOŽE", i "OPCIONALNO" ("MUST", "MUST NOT",
+"REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY",
+"OPTIONAL") u ovom dokumentu treba interpretirati kao što je opisano u
+[RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+1. Softver koji koristi Semantičko verzioniranje MORA objaviti javni API. API
+može biti deklariran u samom kodu ili postojati samo unutar dokumentacije.
+U svakom slučaju, mora biti precizan i jasan.
+
+1. Normalna oznaka verzije MORA biti u formatu X.Y.Z gdje su X, Y i Z prirodni
+(non-negative integer) brojevi i ne smiju počinjati s nulom. X označava
+veću verziju, Y manju verziju, a Z zakrpu.
+Svaki se element MORA numerički povećavati. Na primjer: 1.9.0 -> 1.10.0 -> 1.11.0.
+
+1. Jednom kad je verzionirani paket izdan, sadržaj te verzije NE SMIJE se
+mijenjati. Sve promjene MORAJU se izdavati s novom verzijom.
+
+1. Veća verzija nula (0.y.z) je za inicijalni razvoj. Bilo što se može mijenjati
+u tom stadiju. Javni API se ne smatra stabilnim.
+
+1. Verzija 1.0.0 definira javni API. Način na koji će se oznaka verzije mijenjati
+ovisi o tom javnom APIju i načinu na koji se on mijenja.
+
+1. Oznaka verzije zakrpe Z (x.y.Z | x > 0) MORA se povećavati samo ako se dodaju
+unatrag kompatibilni ispravci bugova. Ispravci bugova definiraju se kao promjene
```

Comment:
I would put "pogrešaka" instead of "bugova".

Reply:
I think we have a similar situation here as with *build* - "pogreška" could also be translated as *typo* or just *error* while bug could (if I'm not mistaken) also be an unexpected behaviour

Reply:
switched to italics